com.regnosys.rosetta.ide/src/com/regnosys/rosetta/ide/highlight/RosettaHighlightingCalculator.xtend

```diff
@@ -25,7 +25,7 @@ import org.eclipse.xtext.nodemodel.util.NodeModelUtils
 import org.eclipse.xtext.util.CancelIndicator
 
 import static com.regnosys.rosetta.rosetta.RosettaPackage.Literals.*
-import com.regnosys.rosetta.rosetta.RosettaDocReference
+import com.regnosys.rosetta.rosetta.RegulatoryDocumentReference
```

Comment:
This is a non-backward compatible change. This means that this release is blocked from being used in Rosetta. All ecore changes as this point must be backward compatible.

Reply:
I take it back. I can see its backwards compatible further in the review.

Reply:
I don't think this kind of change will cause a problem. If I push this through, Rosetta will be able to handle this ecore change.

Or is this related to the BSP work?